packages/node/src/indexer/fetch.service.ts

```diff
@@ -499,8 +504,8 @@ export class FetchService implements OnApplicationShutdown {
               const enqueuingBlocks = batchBlocks.slice(0, maxBlockSize);
               const cleanedBatchBlocks =
                 this.filteredBlockBatch(enqueuingBlocks);
-
-              this.blockDispatcher.enqueueBlocks(
+              // eslint-disable-next-line @typescript-eslint/await-thenable
```

Comment:
Why is this now needed?

Reply:
enqueueBlocks of `WorkerBlocksDispacher` is Thenable while that of `BlockDispatcher` is not. So, the linter will catch that.